flight/targets/aq32/board-info/board_hw_defs.c

```diff
@@ -911,10 +913,9 @@ static const struct pios_sbus_cfg pios_usart3_sbus_aux_cfg = {
 		},
 	},
 	.gpio_inv_enable = Bit_SET,
+	.gpio_inv_disable = Bit_RESET,
 };
 
-#endif	/* PIOS_INCLUDE_SBUS */
```

Comment:
Will move it back; I'm betting it was accidental.

Not a fan of these in the board_hw_defs, etc.  If the object isn't used the linker will omit it, so they're non-functional.